wagtail/embeds/oembed_providers.py

```diff
@@ -199,12 +199,12 @@
 geograph_gg = {
     "endpoint": "http://www.geograph.org.gg/api/oembed",
     "urls": [
-        r"^https?://(?:[-\w]+\.)?geograph\.org\.gg/.+$",
-        r"^https?://(?:[-\w]+\.)?geograph\.org\.je/.+$",
-        r"^https?://channel-islands\.geograph\.org/.+$",
-        r"^https?://channel-islands\.geographs\.org/.+$",
-        r"^https?://(?:[-\w]+\.)?channel\.geographs\.org/.+$",
-    ],
+        r"^http://.*\.geograph\.org\.gg/.*$",
+        r"^http://.*\.geograph\.org\.je/.*$",
+        r"^http://channel-islands\.geograph\.org/.*$",
+        r"^http://channel-islands\.geographs\.org/.*$",
+        r"^http://.*\.channel\.geographs\.org/.*$"
+    ]
```

Comment:
I this these geographs ones would still need https right?
Also, double check you have run the formatting, it looks like the nesting is out here. 

See https://docs.wagtail.org/en/stable/contributing/python_guidelines.html#python-coding-guidelines

Reply:
Thank you for bringing this to my attention. The correct URL should use "https," and I will ensure this is corrected in the next commit.